specialMenu.php

```diff
@@ -15,106 +15,97 @@
 		<div class="accordion-item">
 			<h2 class="accordion-header" id="monday">
 				<button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapseOne" aria-expanded="false" aria-controls="flush-collapseOne">
-					Milk Chocolate Mondays
+					Milk Chocolate Monday
 				</button>
 			</h2>
 			<div id="flush-collapseOne" class="accordion-collapse collapse" aria-labelledby="monday" data-bs-parent="#accordionSpecials">
 				<div class="accordion-body">
-					<p>Come and enjoy hot milk chocolate for a party of 4 for only the price of 3.
-						We'll serve a giant bowl of chocolate and only two spoons so sharing is caring!
-						Milk Chocolate Mondays are a good way to get closer with family and friends. <br />Don't
-						have friends?? Have no fear, matchmaking is here. Come to the counter and ask to
+					<p>Come and enjoy hot milk chocolate for a party of four for only the price of three.
+						The Milk Chocolate Monday package comes with a giant mug of decadent hot chocolate and only two spoons so sharing is caring!
+						Milk Chocolate Monday are a good way to get closer with family and friends. <br />Don't
+						have friends? Have no fear, our StarBulls Matchmaker&trade; is here. Come to the counter and ask to
 						sign up for a matchmaking table where we set you up with people also looking
-						for new friends or family.... ($50/per table of 4)</p>
+						for new friends...or...family... ($50/per table of 4)</p>
 				</div>
 			</div>
 		</div>
 		<div class="accordion-item">
 			<h2 class="accordion-header" id="tuesday">
 				<button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapseTwo" aria-expanded="false" aria-controls="flush-collapseTwo">
-					Tingle Tuesdays
+					Try to Stay Awake Tuesday
```

Comment:
I just saw this... was this always here? Can we do better?

Reply:
yeah but give me some time to rework the content. I'm gonna rewrite this entire one for clarity

Reply:
:roll_eyes: lol ok